build/Linux/Dockerfile_x86_64.dockerfile

```diff
@@ -7,5 +7,6 @@ RUN bash -c 'echo export BOOST_ROOT=/usr/include/boost148 >>~/.bashrc;\
 echo export BOOST_LIB64=/usr/lib64/boost148 >> ~/.bashrc'
 RUN bash -c 'cd /usr/lib64;\
 ln -s libboost_thread-mt.so.1.48.0 libboost_thread.so;\
-ln -s libboost_system-mt.so.1.48.0 libboost_system.so'
+ln -s libboost_system-mt.so.1.48.0 libboost_system.so;\
+ln -s libboost_serialization-mt.so.1.48.0 libboost_serialization.so'
```

Comment:
Hi Sam, wondering why this serialisation thing is needed in the 64-bit Linux build but not the 32-bit.